src/util/worker/mod.rs

```diff
@@ -143,20 +147,22 @@ impl<T: Display> Clone for Scheduler<T> {
 #[cfg(test)]
 pub fn dummy_scheduler<T: Display>() -> Scheduler<T> {
     let (tx, _) = mpsc::channel();
-    Scheduler::new("dummy scheduler", AtomicUsize::new(0), tx)
+    Scheduler::new("dummy scheduler", AtomicUsize::new(0), tx, 1)
 }
 
 #[derive(Copy, Clone)]
 pub struct Builder<S: Into<String>> {
     name: S,
     batch_size: usize,
+    busy_limit: usize,
 }
 
 impl<S: Into<String>> Builder<S> {
     pub fn new(name: S) -> Self {
         Builder {
             name: name,
             batch_size: 1,
+            busy_limit: 1,
```

Comment:
Why default value is 1?

Reply:
Because now `Scheduler::is_busy` returns `true` if `self.counter >= busy_limit`, not `self.counter > busy_limit`. I think `1` is more normal than `0`, just like if we need a sync channel, we need `channel(1)` instead of `channel(0)`.